app/tools/android/make_apk.py

```diff
@@ -142,12 +140,30 @@ def FindExtensionJars(root_path):
         extension_jars.append(extension_jar)
   return extension_jars
 
```

Comment:
Two empty lines between function, you could use PEP8 style checker(ignoring the 4-space indent warnings)